Perlang.Tests/Blocks.cs

```diff
@@ -17,7 +17,7 @@ public void calling_an_undefined_function_inside_a_block_throws_expected_excepti
         }
 
         [Fact]
-        public void referring_to_an_undefined_variable_inside_a_block_throws_expected_exception()
+        void referring_to_an_undefined_variable_inside_a_block_throws_expected_exception()
```

Comment:
Drive-by change. I think we could perhaps start dropping the `public` part for tests. It's not really needed and the test code is arguably a bit more readable without it (less noise when skimming through test classes).